site/src/api/queries/organizations.ts

```diff
@@ -0,0 +1,46 @@
+import type { QueryClient } from "react-query";
+import { API } from "api/api";
+import type {
+  CreateOrganizationRequest,
+  UpdateOrganizationRequest,
+} from "api/typesGenerated";
+import { meKey, myOrganizationsKey } from "./users";
+
+export const createOrganization = (queryClient: QueryClient) => {
+  return {
+    mutationFn: (params: CreateOrganizationRequest) =>
+      API.createOrganization(params),
+
+    onSuccess: async () => {
+      await queryClient.invalidateQueries(meKey);
+      await queryClient.invalidateQueries(myOrganizationsKey);
```

Comment:
When creating an organization, do we add users to it in the same request? If yes, I can understand why we invalidate those queries, if not, I can't understand why.

Reply:
when you create an organization, you are automatically added as the only member.